ebos/ecltransmissibility_impl.hh

```diff
@@ -469,8 +474,8 @@ update(bool global, const std::function<unsigned int(unsigned int)>& map, const
                                                         axisCentroids),
                                         porosity_[outsideElemIdx]);
 
-                applyNtg_(halfDiffusivity1, insideFaceIdx, elemIdx, ntg);
-                applyNtg_(halfDiffusivity2, outsideFaceIdx, outsideElemIdx, ntg);
+                applyNtg_(halfDiffusivity1, insideFaceIdx, coarseElemIdx, ntg);
+                applyNtg_(halfDiffusivity2, outsideFaceIdx, coarseOutsideElemIdx, ntg);
```

Comment:
Does this still work after the change above? I thought it should be elemIdx outsideElemIdx as before.

Reply:
Indeed, thanks for noticing it!